quadra/utils/utils.py

```diff
@@ -312,6 +347,11 @@ def finish(
                                     signature=signature,
                                 )
                         elif model_type in ["onnx", "simplified_onnx"]:
+                            if not ONNX_AVAILABLE:
+                                raise ImportError(
```

Comment:
Error is too much here, we allow setting export.types["onnx"] and during export there is a warning saying "you have to install onnx skipping)

Since there's a warning before I would do

elif model_type in ["onnx", "simplified_onnx"] and ONNX_AVAILABLE: